lib/bloc/account/account_bloc.dart

```diff
@@ -72,36 +72,56 @@ class AccountBloc {
   final _lightningDownController = new StreamController<bool>.broadcast();
   Stream<bool> get lightningDownStream => _lightningDownController.stream;
 
+  final _startLightningController = new StreamController<bool>();
+  Sink<bool> get startLightningSink => _startLightningController.sink;
+
 
   Stream<Map<String, DownloadFileInfo>>  chainBootstrapProgress;
-  BreezUserModel _currentUser;  
+  BreezUserModel _currentUser;
   bool _allowReconnect = true;
+  bool _startedLightning = false;
 
   AccountBloc(Stream<BreezUserModel> userProfileStream) {
       ServiceInjector injector = new ServiceInjector();    
-      BreezBridge breezLib = injector.breezBridge;  
+      BreezBridge breezLib = injector.breezBridge;
       BreezServer server = injector.breezServer;
       Notifications notificationsService = injector.notifications;
-      Device device = injector.device;      
+      Device device = injector.device;
 
       _accountController.add(AccountModel.initial());
       _paymentFilterController.add(PaymentFilterModel.initial());
       //listen streams      
       _listenUserChanges(userProfileStream, breezLib);
       _listenNewAddressRequests(breezLib);
-      _listenWithdrawalRequests(breezLib);  
+      _listenWithdrawalRequests(breezLib);
       _listenSentPayments(breezLib);
       _listenFilterChanges(breezLib);
       _listenAccountChanges(breezLib);
       _listenPOSFundingRequests(server, breezLib);
       _listenMempoolTransactions(device, notificationsService, breezLib);
       _listenRoutingNodeConnectionChanges(breezLib);
-       breezLib.bootstrapAndStart();      
-       _refreshAccount(breezLib);
-       _listenConnectivityChanges(breezLib);   
-       _listenReconnects(breezLib);
-       _listenRefundableDeposits(breezLib, device);
-       _listenRefundBroadcasts(breezLib);
+      _listenStartLightningRequests(breezLib, device);
+    }
+
+    void _startLightning() {
+
+    }
+
+    void _listenStartLightningRequests(BreezBridge breezLib, Device device) {
+      _startLightningController.stream.listen((start) {
```

Comment:
There are four places that pushing to this sink in attempt to start the lightning lib. I think it makes it complicated to track. Perhaps a better way is to just to start the lightningLIb in the account bloc when the user is registered. This I think will cover all the cases (startup with/without restore, startup with already registered user ..)
And it will be a single point of decision without need for the UI to do it from different widgets.
Also it is simple to implement because the account bloc already tracks the user stream.